sidestep_engine/gui/file_ops.py

```
@@ -547,49 +557,59 @@ def create_mix_dataset(
-                    # Sidecar failures should not fail the whole mix.
-                    pass
+                    dest_sidecar.write_text("", encoding="utf-8")
+            else:
+                dest_sidecar.write_text("", encoding="utf-8")
+
+    if created == 0:
+        # Clean up the empty directory we created
+        try:
+            shutil.rmtree(str(out_dir), ignore_errors=True)
+        except Exception:
+            pass
```

Ruff (0.16.0)

[warning] Use `contextlib.suppress(Exception)` instead of `try`-`except`-`pass`

Replace `try`-`except`-`pass` with `with contextlib.suppress(Exception): ...`

(SIM105)